go/types/path.go

```diff
@@ -17,7 +17,10 @@ import (
 	"github.com/attic-labs/noms/go/hash"
 )
 
-var annotationRe = regexp.MustCompile("^([a-z]+)")
+// For an annotation like @type, 1st capture group is the annotation.
+// For @at(42), 1st capture group is the annotation and 3rd is the parameter.
+// Note, @at() is valid under this regexp, code should deal with the error.
```

Comment:
So I can distinguish between `@at` and `@at()` for constructing error messages, i.e. "missing annotation" vs "not a number".